e2e-test/helper.py

```diff
@@ -0,0 +1,198 @@
+
+
+# KICK OFF A WORKFLOW INSIDE A WORKSPACE
+def submit_workflow_assemble_refbased(workspaceId, dataFile, azure_token):
+    cbas_url = poll_for_app_url(workspaceId, "cbas", azure_token)
+    logging.debug(cbas_url)
+    #open text file in read mode
+    text_file = open(dataFile, "r")
+    request_body = text_file.read();
+    text_file.close()
+    
+    cbas_run_sets_api = f"{cbas_url}/api/batch/v1/run_sets"
+    
+    headers = {"Authorization": azure_token,
+               "accept": "application/json", 
+              "Content-Type": "application/json"}
+    
+    response = requests.post(cbas_run_sets_api, data=request_body, headers=headers)
+    # example of what it returns:
+    # {
+    #   "run_set_id": "cdcdc570-f6f3-4425-9404-4d70cd74ce2a",
+    #   "runs": [
+    #     {
+    #       "run_id": "0a72f308-4931-436b-bbfe-55856f7c1a39",
+    #       "state": "UNKNOWN",
+    #       "errors": "null"
+    #     },
+    #     {
+    #       "run_id": "eb400221-efd7-4e1a-90c9-952f32a10b60",
+    #       "state": "UNKNOWN",
+    #       "errors": "null"
+    #     }
+    #   ],
+    #   "state": "RUNNING"
+    # }
+    logging.debug(response.json())
+
+def clone_workspace(billing_project_name, workspace_name, header):
+    clone_workspace_api = f"{rawls_url}/api/workspaces/{billing_project_name}/{workspace_name}/clone";
+    request_body = {
+        "namespace": billing_project_name,  # Billing project name
+        "name": f"{workspace_name} clone-{''.join(random.choices(string.ascii_lowercase, k=3))}",  # workspace name
+        "attributes": {}};
+
+    logging.info(f"cloning workspace {workspace_name}")
+    response = requests.post(url=clone_workspace_api, json=request_body, headers=header)
+    assert response.status_code == 201, f"Cloning {workspace_name} failed: {response.reason}"
+    # example json that is returned by request: 'attributes': {}, 'authorizationDomain': [], 'bucketName': '', 'createdBy': 'yulialovesterra@gmail.com', 'createdDate': '2023-08-03T20:10:59.116Z', 'googleProject': '', 'isLocked': False, 'lastModified': '2023-08-03T20:10:59.116Z', 'name': 'api-workspace-1', 'namespace': 'yuliadub-test2', 'workspaceId': 'ac466322-2325-4f57-895d-fdd6c3f8c7ad', 'workspaceType': 'mc', 'workspaceVersion': 'v2'}
+    clone_response_json = response.json()
+    logging.debug(clone_response_json)
+    return clone_response_json["workspaceId"]
+
+def check_wds_data(wds_url, workspaceId, recordName, azure_token):
+    version = "v0.2"
+    api_client = wds_client.ApiClient(header_name='Authorization', header_value="Bearer " + azure_token)
+    api_client.configuration.host = wds_url
+
+    schema_client = wds_client.SchemaApi(api_client)
+
+    logging.info("verifying data was cloned")
+    response = schema_client.describe_record_type(workspaceId, version, recordName);
+    assert response.name == recordName, "Name does not match"
+    assert response.count == 2504, "Count does not match"
+
+
+def delete_workspace():

```

Comment:
since you are running this and then destroying the BEE - any reason to keep this in here? (its not a required to do for this per say)